apps/react/src/challenges/transfer-list/Components/Buttons/Buttons.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+const Buttons = ({
+  leftClickHandler,
+  rightClickHandler,
+  leftClickHandlerAll,
+  rightClickHandlerAll,
+  disableLeftBtn,
+  disableRightBtn,
+  disableLeftBtnAll,
+  disableRightBtnAll,
+}) => {
+  
+  return (
+    <div className="buttons-container">
+      <button
+        onClick={() => leftClickHandlerAll()}
+        disabled={disableLeftBtnAll}
+        style={{ pointerEvents: disableLeftBtnAll ? "none" : "" }}
```

Comment:
can we have class to disabled button instead of having style on each

Reply:
Done